sleap/nn/evals.py

```diff
@@ -203,11 +209,20 @@ def compute_oks(
     assert distance.shape == (n_gt, n_pr, n_nodes)
 
     # Compute the normalization factor per keypoint.
-    spread_factor = stddev ** 2
-    scale_factor = 2 * (scale + np.spacing(1))
-    normalization_factor = np.reshape(spread_factor, (1, 1, n_nodes)) * np.reshape(
-        scale_factor, (n_gt, 1, 1)
-    )
+    # If use_cocoeval is False, then compute normalization factor according to the paper.
+    if use_cocoeval == False:
+        spread_factor = stddev ** 2
+        scale_factor = 2 * ((scale + np.spacing(1)) ** 2)
+        normalization_factor = np.reshape(spread_factor, (1, 1, n_nodes)) * np.reshape(
+            scale_factor, (n_gt, 1, 1)
+        )
+    # If use_cocoeval is True, then compute normalization factor according to cocoeval.
+    else:
+        spread_factor = (2 * stddev) ** 2
+        scale_factor = 2 * (scale + np.spacing(1))
+        normalization_factor = np.reshape(spread_factor, (1, 1, n_nodes)) * np.reshape(
+            scale_factor, (n_gt, 1, 1)
+        )
```

Comment:
```suggestion
    if use_cocoeval:
        # If use_cocoeval is True, then compute normalization factor according to cocoeval.
        spread_factor = (2 * stddev) ** 2
        scale_factor = 2 * (scale + np.spacing(1))
        normalization_factor = np.reshape(spread_factor, (1, 1, n_nodes)) * np.reshape(
            scale_factor, (n_gt, 1, 1)
        )
    else:
        # If use_cocoeval is False, then compute normalization factor according to the paper.
        spread_factor = stddev ** 2
        scale_factor = 2 * ((scale + np.spacing(1)) ** 2)
        normalization_factor = np.reshape(spread_factor, (1, 1, n_nodes)) * np.reshape(
            scale_factor, (n_gt, 1, 1)
        )
```

It should be pretty safe to use the bool stored in `use_cocoeval` for conditionals.